baseplate/frameworks/pyramid/__init__.py

```diff
@@ -124,21 +124,22 @@ def should_trust_edge_context_payload(self, request: Request) -> bool:
 
 # pylint: disable=too-many-ancestors
 class BaseplateRequest(RequestContext, pyramid.request.Request):
-    def __init__(self, context_config: Dict[str, Any], environ: Dict[str, str]):
-        RequestContext.__init__(self, context_config)
-        pyramid.request.Request.__init__(self, environ)
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
I'm curious why you changed the function signature. It _seems_ like the original signature would have worked.